ProjFS.Mac/PrjFSLib/PrjFSLib.cpp

```diff
@@ -331,6 +331,7 @@ PrjFS_Result PrjFS_WritePlaceholderFile(
         goto CleanupAndFail;
     }
     
+    // TODO(Mac): Only call chmod is fileMode is different than the default file mode
```

Comment:
Good call

Reply:
I see I made a typo in this comment, I'll fix that up